packages/coreutils/src/settingregistry.ts

```diff
@@ -20,8 +20,8 @@ import {
 } from '@phosphor/signaling';
 
 import {
-  IDataConnector, StateDB
-} from '.';
```

Comment:
Good point. I thought it was "cute", which is probably a good reason to not do it.